src/app/services/age-verification/age-verification.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AgeVerificationService {
+  readonly LEGAL_AGE = 18;
+
+  ageVerification(age: number, firstName: string, lastname: string): string {
```

Comment:
In clean code concepts you cant have more than 2 parameters. A good approach is applyin an object, example:

```typescript
ageVerification({ age, firstName, lastName }): string {
```

Reply:
Do not forget to add an interface there:



```
interface IUser {
  age: number;
  firstName: string;
  lastname: string;
}

ageVerification({ age, firstName, lastName }: IUser): string {
```

Reply:
should all parameters have an Interface?

should all function have the type that returns?

This is to add the answers to my notes

Reply:
Basically everything must be an interface because you are not working alone ;)

Everything should have return types except when is "void", you can omit